src/lib.rs

```diff
@@ -2,7 +2,7 @@
 #![deny(missing_debug_implementations)]
 #![deny(missing_docs)]
 #![no_std]
-#![cfg_attr(feature = "allocator_api", feature(allocator_api))]
+#![cfg_attr(feature = "nightly", feature(allocator_api))]
```

Comment:
Ideally, the API and cargo features for enabling the nightly allocator API support in `bumpalo` wouldn't change just because we can alternatively use the `allocator_api2` crate, so the "nightly" feature should keep the "allocator_api" name.